lib/awesome_print/formatters/ext/sequel_model_class_formatter.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../base_formatter'
+
+module AwesomePrint
+  module Formatters
+    class SequelModelClassFormatter < BaseFormatter
+
+      formatter_for :sequel_model_class
+
+      def self.formattable?(object)
+        defined?(::Sequel::Model) && object.is_a?(Class) && object.ancestors.include?(::Sequel::Model)
+      end
+
+      def format(object)
+        data = object.db_schema.inject({}) { |h, (prop, defn)| h.merge(prop => defn[:db_type]) }
```

Comment:
With isolated state like `inject` or `reduce` it may make more sense to use `merge!` to avoid extra object allocations